### PR TITLE
Spring Boot Kotlin example

### PR DESCRIPTION
**Pull request by chrisime**

### CODE REVIEW

File: settings.gradle [hunk 1]
```diff
@@ -1,6 +1,6 @@
 rootProject.name = 'fw-monitoring-examples'
 include 'ktor-monitoring'
 include 'spring-boot-monitoring'
+include 'spring-boot-monitoring-kt'
```
Comment (mirage22) [inline settings.gradle]: is possible to follow proposed naming "framework"-"language"-"purpose", example spring-boot-kt-monitoring ?

**Review by mirage22 — CHANGES_REQUESTED** [review]

**Comment by chrisime** [comment]
@mirage22 all done

File: buildMonitoringContainers.sh [hunk 2]
```diff
@@ -1,10 +1,10 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
```
Comment (mirage22) [inline buildMonitoringContainers.sh]: is possible to stay with sh ?

**Review by mirage22 — CHANGES_REQUESTED** [review]

File: spring-boot-kotlin-monitoring/build.gradle.kts [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+}
+
+repositories {
+    mavenCentral()
+}
+
+tasks.withType<KotlinCompile> {
+    kotlinOptions {
+        freeCompilerArgs = listOf("-Xjsr305=strict")
+        jvmTarget = "16"
```
Comment (mirage22) [inline spring-boot-kotlin-monitoring/build.gradle.kts]: is that possible as variable ? java_version = 16

**Review by mirage22 — CHANGES_REQUESTED** [review]

File: spring-boot-kotlin-monitoring/build.gradle.kts (hunk 3 above)
Comment (chrisime) [reply spring-boot-kotlin-monitoring/build.gradle.kts]: ok

**Review by mirage22 — APPROVED** [review]

**Pull request merged**